input/pagecontent/reference.md

```diff
@@ -668,12 +551,12 @@ Assignment rules follow this syntax:
 * <element> = {value}
 ```
 
-The left side of this expression follows the [FSH path grammar](#fsh-paths). The data type on the right side must align with the data type of the final element in the path, and may be a complex data type (such as an address).
+The left side of this expression follows the [FSH path grammar](#fsh-paths). The data type on the right side must align with the data type of the final element in the path, and may be a primitive or complex data type or a resource reference.
```

Comment:
I think this is a bit confusingly worded. What do you mean by resource reference here? Do you mean the `Reference` keyword, or just the name of an Instance.

Reply:
Fixed.